checklists/acr_security_checklist.en.json

```diff
@@ -0,0 +1,220 @@
+      "description": "Azure Container Registry automatically encrypts images and other artifacts that you store. By default, Azure automatically encrypts the registry content at rest by using service-managed keys. By using a customer-managed key, you can supplement default encryption with an additional encryption layer.",
+      "guid": "0bd05dc2-efd5-4d76-8d41-d2500cc47b49",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/container-registry/tutorial-customer-managed-keys"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Identity and Access Control",
+      "text": "Disable Azure ARM audience tokens for authentication",
+      "description": "Only tokens with an ACR audience can be used for authentication. Used when anabling Conditional access policies for ACR",
+      "guid": "3a041fd3-2947-498b-8288-b3c6a56ceb54",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/container-registry/container-registry-enable-conditional-access-policy"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Vulnerability Management",
+      "text": "Deploy validated container images",
+      "description": "Deploy trusted code that was validated and scanned for vulnerabilities according to DevSecOps practices.",
+      "guid": "4451e1a2-d345-4293-a763-9637a551c5c0",
+      "severity": "Medium"
+    },
+    {
+      "category": "Security",
+      "subcategory": "Vulnerability Management",
+      "text": "Use up-to-date platforms, languages, protocols and frameworks",
+      "description": "Use the latest versions of supported platforms, programming languages, protocols, and frameworks.",
+      "guid": "4e401955-387e-45ce-b126-cd132af5b20c",
+      "severity": "High"
+    }
+  ],
+  "categories": [
+    {
+      "name": "Identity and Access Management"
+    },
+    {
+      "name": "Network Topology and Connectivity"
+    },
+    {
+      "name": "Business Continuity and Disaster Recovery"
+    },
+    {
+      "name": "Security, Governance, and Compliance"
+    },
+    {
+      "name": "Application Automation and DevOps"
+    },
+    {
+      "name": "Management and Monitoring"
+    }
+  ],
+  "status": [
+    {
+      "name": "Not verified",
+      "description": "This check has not been looked at yet"
+    },
+    {
+      "name": "Open",
+      "description": "There is an action item associated to this check"
+    },
+    {
+      "name": "Fulfilled",
+      "description": "This check has been verified, and there are no further action items associated to it"
+    },
+    {
+      "name": "N/A",
+      "description": "Not applicable for current design"
+    },
+    {
+      "name": "Not required",
+      "description": "Not required"
+    }
+  ],
+  "severities": [
+    {
+      "name": "High"
+    },
+    {
+      "name": "Medium"
+    },
+    {
+      "name": "Low"
+    }
+  ],
+  "metadata": {
+    "name": "Azure App Security Review"

```

Comment:
Can it be that the name of the checklist is wrong? Should it be something like "Azure Container Registry Security Review"?

Reply:
@erjosito, well spotted, I'll change it now...